steps/magicpod-ui-test/assets/icon.svg

```diff
@@ -0,0 +1,3 @@
nXjwuNx739+jVqaxNSJ7TK8XM39rqw6/2d6NCABEFQsBQ7pQlo7XjRtCfWW7kolZms/Nkp58sPH8Mb+p0j4LyDhr1E9g8Nl/eItj93LSSDC/MNVfll113MJ/AX3dJrUWEC4p+SlRiaBgyW7lYmsurCf/WF7gWSfSfgVbfdVUf0ylp53dVv9DotJwT9veY0mN7+zKWBNyZ2GTT09xOttWI1JANTIuaFz4kbv7x3Qxvws/NnFWfjb6kfxedYapPhP67fCby9fMy2PUYguk8leXmrirBWw8c37m96mcN0VmJAwCTNHX9CGCRMAx+rjFQd2n1XhkdTV/ecPExtP+I7xW4C/fHGXinz044tvQ4hfqKprPQfEWh7jwUupXrR6wRGKddvoP6U985waigC44+cxbQPFhONIMVzpWr3lSucQWV/s/gxPfvYIaim0VjyZjvbXnt/eJOxWj77u0eSgZCcAJjie/V722i+wtWADwrwScdfW3+Mzl404b9T5Cg/GgtfROR5g/05kf0HazzC/yfjo0Ns4WLgHP531S0whF27WetSEHzcK6gt4MpS1BnfaHdhIiejdQIl6ew+KclveUEyGKuWq4NzbcYPflb4D97y6DHd+eDncabuvQFOICh6p1eP+iBLrONyoecnO2+TTVoT0XPKqK3gTo4NmItQSiQt9pmLD3rX2CbBWzYjDh/PyXX8uvy4wbwfOkYDZQOmnqxbjL2//SXkW2ldL7CLAhk2lFBKO7TGcRoj6NgPyaDgNgMf5uRSDn+PGRdEa8J5ju7B+51q8tvdpWv+OVuN9nknmZbKB0PhZFWbSc9y5uJH23mPHOS6jtmng1Q5Wf8kYQCU2EKJd+gZIo2f7jwa1rDnM9xysObgKn6R9iGvH/ViFgGOPzbIqMm6izUkG0S5EpAYNkHJ3XAxDEQBPeZGiR1t1heGfG55Rbr1rM16lnWpCkOA9moSCmnyryW7H0PX9M7gZ88YiJQ25NKlZ1ZZh3ovQmwSdJwSZ5Ng2wuLpreL565M4vp6nCxsfDRxhYJW/gfZIiCATbH7/2Nf34OVtgzAj7iLk005GseSApAlR4zDQjwYjAG70ZAtOO+UcLtqv3Ep5ptg+YTRQejvHJssbnrihlHY8clzyYqOXII8hSgh4+MPDBJ4j0BoP41HTUE2aA8+KD7zEhMdlHeo7gcivCZ+n/1ftdcB7RXLZjZQMRwBcuUwCvEEEN4KBzvg8i1/RSPb7ZO2oUwR5HMZahqGWvmfLueLGw0gKHapmw/kcNiXmjTvN5FegJsr0hQPoyILOwUS4DQR5hbERtdKIBlARO1UUY00COkDCDZutvgZ64mFNOMXMyyhObytqIIU5iw8eQh6I1co4qNm15gTX6JKqYhwp2W8Pk0U4DeTERMAakN0+YiCXtP2yGZYA2odj4H3LjTuYfB92H9+hvP24hLwUOCVxGopov0FeKx8XtFjFx9OlzyrIpH0SvlHbpxtNJdYYGOUoBDDAa5qdefwoss63eZ/Q0td3WsCYpLGIMCchtyYD5yXNVht1apuIbw9/DR+XaKUWD3B4DF88IYAB3wTY8s+NNg8tUi7NXFwW9MFh8ViQshRf1+7D7LFzFAo8Ici7FG888jEiyGeeXYglDWwEhAAGdv2q0rGf/CDzSAplvpbCnle3mbpePG4eXjr/n0iISFTej3zylkObcKjyS7VmLur/wG8chlwFGPjVemIJecKT4xJuyd+A7Ue3YfqIGUrgE6OSwC9e62eLyHIK6/XOtteRaJmqlkhPvIt8GogIiAYwEGu1nTLxZGA42f3/68vnlAMQC7we89OIQKX129dhe9EnKmbhQF3+awcaQ38lBGCQ6meB9qV4g5vy12Ld1o/aSs3f89g/qzADz339VwpbPnHA20a0FV7e0DKwJMMgwGbOyb6T8fRXf0Z63lH7XEBr77/qq9fJMKhWDQVk7G+YJiEEYJyq/q6ktHMhnln3hHJ/5qHA2q1r8Nqep1XIMzaTlWQcBGQS0Dh1rUrKcwGBtDnGf8kJauwX48kAiLZGX78cseYRovobrC1wcQ29PbgB61sVmVV8DwrtXVCfjSqK7R9vGdP2vVExMWq5RQMwYM2zAwxvuRVuikEEaLNNUvtl3G/AhkBFFgIwZr2rUst438CV31p0WQWQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGAEhAANXvhRdEBACkDYgCBgYASEAA1e+FF0QEAKQNiAIGBgBIQADV74UXRAQApA2IAgYGIH/D+bwh2c+vVZwAAAAAElFTkSuQmCC" x="0" y="0" width="256" height="256" transform="matrix(1 0 0 1 0 0)"></image></g></g></g></g></svg>
```

Comment:
I would say that icon file should use vector format rather than just being a wrapper around PNG (like all official steps).